storybook/stories/radar/Radar.stories.tsx

```diff
@@ -227,3 +227,14 @@ export const WithPatterns: Story = {
 export const WithRotation: Story = {
     render: args => <Radar {...commonProperties} curve={args.curve} rotation={36} />,
 }
+
```

Comment:
I think it can be removed as it's now included to the website, less to maintain :)